shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui25/components/SwitcherItem.kt

```diff
@@ -0,0 +1,80 @@
+package org.jetbrains.kotlinconf.ui25.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import org.jetbrains.compose.ui.tooling.preview.Preview
+import org.jetbrains.kotlinconf.ui25.theme.KotlinConfTheme
+import org.jetbrains.kotlinconf.ui25.theme.PreviewHelper
+
+
+enum class SwitcherItemState {
+    Active, Normal, Selected,
+}
+
+@Composable
+fun SwitcherItem(
```

Comment:
Could you be also more specific with naming? If we have a second switcher, we could struggle to distinguish between them

Reply:
This is the name of the component in the design.

![image](https://github.com/user-attachments/assets/16872c3c-3390-4cb4-acf8-e0933abc9190)

Reply:
lgtm